File: problems/problems_1625/Solution.cpp
//go:build ignore
#include "cpp/common/Solution.h"


using namespace std;
using json = nlohmann::json;

class Solution {
public:
    string findLexSmallestString(string s, int a, int b) {
        
    }
};

json leetcode::qubh::Solve(string input_json_values) {
	vector<string> inputArray;
	size_t pos = input_json_values.find('\n');
	while (pos != string::npos) {
		inputArray.push_back(input_json_values.substr(0, pos));
		input_json_values = input_json_values.substr(pos + 1);
		pos = input_json_values.find('\n');
	}
	inputArray.push_back(input_json_values);

	Solution solution;
	string s = json::parse(inputArray.at(0));
	int a = json::parse(inputArray.at(1));
	int b = json::parse(inputArray.at(2));
	return solution.findLexSmallestString(s, a, b);
}

File: problems/problems_1625/solution.go
package problem1625

import (
	"encoding/json"
	"log"
	"strings"
)

func findLexSmallestString(s string, a int, b int) string {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var s string
	var a int
	var b int

	if err := json.Unmarshal([]byte(inputValues[0]), &s); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &a); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[2]), &b); err != nil {
		log.Fatal(err)
	}

	return findLexSmallestString(s, a, b)
}
